homework/css/index.css

```diff
@@ -0,0 +1,51 @@
+body {
+  background-color: #e2fff4;
+  display: grid;
+  grid-gap: 10px;
+  color: white;
+  font-family: 'Fjalla One', sans-serif;
+  position: relative;
+}
+
+body ul {
```

Comment:
Avoid using tag names as selectors in css. Better add apropriate classes as it will be easier and faster to parse the css code and build the page for browser.

Reply:
I've adjusted some places where it was possible in my point of view.
I mean that the approach that you purpose is better, in future i will follow your advice.